auth/authenticate.js

```diff
@@ -27,3 +30,16 @@ function authenticate(req, res, next) {
     });
   }
 }
+
+function generateToken(user) {
+  console.log('i am superman');
+	const payload = {
+	  subject: user.id, // sub in payload is what the token is about
+	  username: user.username,
+	  context : user.context
```

Comment:
user.context is undefined, the user table only has username and password, besides the id as the primary key.